### PR TITLE
added clipboard buffer url launcher

### PR DESCRIPTION
**Pull request by dcdigit**
## Description of changes (updated or new workflows)
New workflow to take a list of URLs in the clipboard buffer and launch them all at once.

### CODE REVIEW

File: specs/shell/launch-clipboard-buffer-url-list.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Launch tabs with clipboard URLs
+command: "pbpaste | xargs -n1 -I{} open {}"
+tags:
+  - pbpaste
+  - xargs
+  - macosx
+  - clipboard
+  - urls
```
Comment (alokedesai) [inline specs/shell/launch-clipboard-buffer-url-list.yaml:9]: nit: we recommend fewer tags (Tags are visible in Warp in the workflows menu, so the more tags we have the more cluttered the UI is).

Thoughts on just including `pbpaste` here?

File: specs/shell/launch-clipboard-buffer-url-list.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+description: Launch all URLs (arranged in a list in the clipboard buffer). Will be launched using default browser and tab creation preference. Useful for copying a column of URLs from a spreadsheet and visiting them all at once.
+author: DC
```
Comment (alokedesai) [inline specs/shell/launch-clipboard-buffer-url-list.yaml:11]: could you use your github username and URL? If you're not comfortable with that also feel free to remove both fields entirely

**Review by alokedesai — COMMENTED** [review]
Looks great! Just a few small comments

File: specs/shell/launch-clipboard-buffer-url-list.yaml (hunk 1 above)
Comment (dcdigit) [reply specs/shell/launch-clipboard-buffer-url-list.yaml:9]: no problem. pbpaste it is!

**Review by alokedesai — APPROVED** [review]
LGTM!

**Pull request merged**